<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BIM Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>This dialog will help you to set FreeCAD up for efficient BIM workflow, by setting a couple of typical FreeCAD options. This dialog can be accessed again anytime from menu Utils, and more options are available under menu Edit -&gt; Preferences.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="13" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Open a new document at startup</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Number of decimals</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QFontComboBox" name="settingFont">
       <property name="toolTip">
        <string>Default font</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="settingWP">
       <property name="toolTip">
        <string>Where the grid appears at FreeCAD startup</string>
       </property>
       <item>
        <property name="text">
         <string>Auto (adapts to view)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Top (XY)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Front (XZ)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Side (YZ)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Preferred working units</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="settingLinewidth">
       <property name="toolTip">
        <string>Default line width</string>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Default size of a grid square</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="settingDimstyle">
       <property name="toolTip">
        <string>Default dimension style</string>
       </property>
       <item>
        <property name="text">
         <string>dot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>arrow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>slash</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Default grid position</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="settingAuthor">
       <property name="toolTip">
        <string>Your name (optional). You can also add your email like this: John Doe &lt;john@doe.com&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="settingUnits">
       <property name="toolTip">
        <string>The unit you prefer to work with, that will be used in dialogs and measurements. It doesn't prevent you from using other units when entering distances, though</string>
       </property>
       <item>
        <property name="text">
         <string>millimeters</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>centimeters</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>meters</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>inches</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>architectural</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="settingWorkbench">
       <property name="toolTip">
        <string>Check this to make FreeCAD start directly in the BIM workbench</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Default author for new files</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Default text size</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Defaut font</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Default line width</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Default dimension style</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Make BIM the default workbench</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="settingNewdocument">
       <property name="toolTip">
        <string>Check this to make FreeCAD start with a new blank document</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Number of backup files</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QSpinBox" name="settingBackupfiles">
       <property name="toolTip">
        <string>The number of backup files to keep when saving a file</string>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Default dimension arrow size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="settingDecimals">
       <property name="toolTip">
        <string>The number of decimals you wish to see used in the interface controls and measurements</string>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="settingLicense">
       <property name="toolTip">
        <string>Optional license you wish to use for new files. Keep &quot;All rights reserved&quot; if you don't wish to use any particular license</string>
       </property>
       <item>
        <property name="text">
         <string>All rights reserved</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CC-BY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CC-BY-SA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CC-BY-NC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CC-BY-SA-NC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Default license for new files</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Gui::InputField" name="settingGrid">
       <property name="toolTip">
        <string>This is the size of the smallest square of the grid</string>
       </property>
       <property name="text">
        <string>1 m</string>
       </property>
       <property name="unit" stdset="0">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="Gui::InputField" name="settingText">
       <property name="toolTip">
        <string>The default size of texts and dimension texts</string>
       </property>
       <property name="text">
        <string>2 cm</string>
       </property>
       <property name="unit" stdset="0">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="Gui::InputField" name="settingArrowsize">
       <property name="toolTip">
        <string>Default dimension arrow size</string>
       </property>
       <property name="text">
        <string>7.5 mm</string>
       </property>
       <property name="unit" stdset="0">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Main grid line every</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="toolTip">
        <string>How many small squares between each main line of the grid</string>
       </property>
       <property name="suffix">
        <string> square(s)</string>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>You might also want to set the appropriate snapping modes on the Snappng toolbar. Enabling only the snap positions that you need will make drawing in FreeCAD considerably faster.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::InputField</class>
   <extends>QLineEdit</extends>
   <header>Gui/InputField.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
